### PR TITLE
Update release-notes-draft.md

### PR DESCRIPTION
**Pull request by calebamiles**
add link to known issues when using rkt runtime

### CODE REVIEW

**Comment by idvoretskyi** [comment]
LGTM. +1 to @spiffxp for now.

**Comment by dchen1107** [comment]
Please update PR with docker runtime known issue:

Docker runtime Known Issues

 * Kernel crash with Aufs storage driver on Debian Jessie (#27885)

 * Consider running the new kubernetes node problem detector to identify this (and other) kernel issues automatically.
 * File descriptors are leaked in docker v1.11 (#275)

 * Additional memory overhead per container in docker v1.11 (#21737)
* Corrupted docker image layer in docker releases (https://github.com/docker/docker/issues/23184)
 * List of upstream fixes for docker v1.10 identified by RedHat

**Comment by saad-ali** [comment]
Waiting for this to merge before I kick off the official release for 1.5.0

**Comment by dims** [comment]
LGTM 👍

**Comment by saad-ali** [comment]
Let's make sure @dchen1107 comments are addressed.

**Comment by calebamiles** [comment]
updated docker runtime known issues and changed the wording a bit @dchen1107. PTAL

File: release-1.5/release-notes-draft.md [hunk 1]
```diff
@@ -128,6 +128,16 @@ release [38537](https://github.com/kubernetes/kubernetes/issues/38537)
 
 Continuous integration builds have used the following versions of external dependencies, however, this is not a strong recommendation and users should consult an appropriate installation or upgrade guide before deciding what versions of etcd, docker or rkt to use. 
 
-* Docker version 1.11.2, Docker version 1.12.0 [has also been validated](https://github.com/kubernetes/kubernetes/issues/28698)
+* Docker version 1.11.2
```
Comment (dchen1107) [inline release-1.5/release-notes-draft.md]: Can we change this to something like:

We performed a fully validation with the follow docker releases: docker 1.9.1, docker 1.10.3, docker 1.11.2. We also validated docker 1.12.3 through Kubernetes docker automated validation framework.

Comment (dchen1107) [inline release-1.5/release-notes-draft.md]: Or we can re-use what we had for 1.4 release:

If you are using Kubernetes to manage docker containers, please be aware Kubernetes has been validated to work with docker 1.9.1, docker 1.11.2 (#23397), and docker 1.12.0 (#28698) here following with issues for each release.

**Comment by dchen1107** [comment]
LGTM

**Comment by saad-ali** [comment]
Merged proceeding with release

**Pull request merged**